docs/data/pages.ts

```diff
@@ -3,23 +3,69 @@ import type { MuiPage } from '@mui/monorepo/docs/src/MuiPage';
 const pages: MuiPage[] = [
   {
     pathname: '/toolpad/getting-started',
-    title: 'Getting started',
-    icon: 'CreateIcon',
+    icon: 'DescriptionIcon',
     children: [
-      { pathname: '/toolpad/getting-started/setup' },
-      { pathname: '/toolpad/getting-started/tutorial', title: 'Tutorial' },
+      { pathname: '/toolpad/getting-started/overview' },
+      { pathname: '/toolpad/getting-started/quickstart' },
     ],
   },
   {
-    pathname: '/toolpad/data-fetching',
+    pathname: '/toolpad/connecting-to-datasources',
     icon: 'TableViewIcon',
     children: [
-      { pathname: '/toolpad/data-fetching', title: 'Overview' },
-      { pathname: '/toolpad/data-fetching/rest', title: 'REST API' },
-      { pathname: '/toolpad/data-fetching/google-sheets', title: 'Google Sheets' },
-      { pathname: '/toolpad/data-fetching/queries' },
+      {
+        pathname: '/toolpad/connecting-to-datasources/queries',
+      },
+      {
+        pathname: '/toolpad/connecting-to-datasources/connections',
+      },
+      {
+        pathname: '/toolpad/connecting-to-datasources/fetch',
+        title: 'Fetch datasource',
+      },
+      {
+        pathname: '/toolpad/connecting-to-datasources/function',
+        title: 'Function datasource',
+      },
+      {
+        pathname: '/toolpad/connecting-to-datasources/google-sheets',
+        title: 'Google sheets datasource',
+      },
+      {
+        pathname: '/toolpad/connecting-to-datasources/postgreSQL',
```

Comment:
We don't allow upper case in our URLs, some something like this:

```suggestion
        pathname: '/toolpad/connecting-to-datasources/postgresql',
```